src/emmc.c

```diff
@@ -706,6 +708,29 @@ pu_emmc_parse_partitions(PuEmmc *emmc,
             return FALSE;
         }
 
+        if (first_partition) {
+            first_partition = FALSE;
+            PedSector min_offset = 1;
+
+            if (g_str_equal(emmc->disktype->name, "msdos"))
+                min_offset = PARTITION_TABLE_SIZE_MSDOS;
```

Comment:
Initialize `min_offset` with `PARTITION_TABLE_SIZE_MSDOS` as `msdos` is the default partition table.